Cod3rsGrowth.Teste/TesteServicoFilme.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Cod3rsGrowth.Infra;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Cod3rsGrowth.Dominio;
+using Cod3rsGrowth.Servicos.Servicos;
+using Cod3rsGrowth.Servicos.Interfaces;
+
+namespace Cod3rsGrowth.Teste;
+
+public class TesteServicoFilme : TesteBase
+{
+    [Fact]
+    public void EncontrarFilmePorId_DeveRetornarFilme_QuandoFilmeExiste()
+    {
```

Comment:
como conversamos remover o teste e tudo que for referente a ele, deixar pra fazer apenas quando chegar a task certa